README.md

```diff
@@ -2,6 +2,10 @@
 
 This repository hosts the client iCommands, the default command line interface to iRODS.
 
+The client iCommands are released as a single package called `irods-icommands` in the packages.irods.org repository: https://packages.irods.org/
```

Comment:
"in the packages.irods.org repository" -> "in the repository at"

Reply:
Done